src/Components/Routes.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
```

Comment:
Remove this file and put back the routes in the **App.jsx** hard-coded. There is no need to separate them into their corresponding file.

Reply:
Feedback implemented